scripts/add_electricity.py

```diff
@@ -138,8 +138,24 @@ def load_costs(tech_costs, config, elec_config, Nyears=1):
     """
     Set all asset costs and other parameters.
     """
+    if os.path.basename(os.getcwd()) == "pypsa-earth":
+        cooking = os.path.join(
+            os.getcwd(), "resources", config["clean_cooking"]["cooking_costs"]
+        )
+
+    else:
+        script_directory = os.path.dirname(os.path.abspath(__file__))
+        root_directory = os.path.abspath(os.path.join(script_directory, ".."))
+        cooking = os.path.join(
+            root_directory, "resources", config["clean_cooking"]["cooking_costs"]
+        )
+
+    cooking = pd.read_csv(cooking).set_index(["technology", "parameter"]).sort_index()
+
     costs = pd.read_csv(tech_costs, index_col=["technology", "parameter"]).sort_index()
 
+    costs = pd.concat([costs, cooking], ignore_index=False)
```

Comment:
File reads should happen at the bottom of the script. Please see ALL other examples. 
You pass then only a variable over the function e.g. like tech_costs.
Maybe worth considering is just to append the clean cooking tech costs to the tech_costs